Makefile

```diff
@@ -1,12 +1,12 @@
 cluster.opensearch.build:
-	docker-compose --project-directory .ci/opensearch build;
+	docker compose --project-directory .ci/opensearch build;
 
 cluster.opensearch.start:
-	docker-compose --project-directory .ci/opensearch up -d ;
-	sleep 60;
+	docker compose --project-directory .ci/opensearch up -d ;
+	sleep 90;
```

Comment:
Replace this with an active check? 

You can do a health check with https://github.com/opensearch-project/opensearch-benchmark/blob/f4ab3ab353ed2826e6c774f231878b387d2ab1c9/osbenchmark/resources/docker-compose.yml.j2#L41 or https://github.com/opensearch-project/dashboards-observability/blob/69d12cee1b0e43869d078e30cc996e9d685e44a9/docker-compose.yml#L47 maybe. Can be added in a future PR.